src/components/card/ProductCard.jsx

```diff
@@ -50,14 +57,14 @@ return <div key={_id}>
                         ({ discount }% OFF)
                     </h4>
                 </span>
-
             </div>
-            <button className="btn btn-shop card-fix-btn card-btn-pri ">
-                <FaShoppingCart /> Add to Cart </button>
-
+            { (authState.loginStatus && authState.cart.some((item)=>item._id===_id)) ? (<button className="btn btn-shop card-fix-btn card-btn-sec" onClick={()=>navigator("/cart") }  >
```

Comment:
Suggestion: You can destrucure `authState` above

Reply:
yeah , it will be easier